lib/mirage_block_partition.mli

```diff
@@ -17,4 +17,7 @@ module Make(B : Mirage_block.S) : sig
   val get_offset : t -> int64
   (** [get_offset b] is the sector offset of the partition relative to the
       underlying block device. *)
-end
+
+  val connect' : int -> t
+   (* [connect'] takes an offset and a length and returns a single partition. *)
```

Comment:
The signature doesn't match the description. The description says it takes an offset and a length, but the signature describes a function that takes a single argument.

Reply:
Yes, now it matches the description. However, it doesn't match with the function you sent a screenshot of above.